src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java

```diff
@@ -135,4 +140,19 @@
 		return optionalTask.map(ControllerResult::of).orElseGet(() -> ControllerResult.of(Error.NOT_FOUND, NAME_KEY));
 	}
 
+	public ControllerResult<GetMarkTaskAsDoneResponseDTO> markTaskAsDone(int taskId, Date date) {
+		Optional<TaskModel> optionalTask = this.taskRepository.getTaskByTaskId(taskId);
+		if (!optionalTask.isPresent()) return ControllerResult.of(Error.NOT_FOUND, NAME_KEY);
```

Comment:
make a new static class in Constants TaskDone and use Constants.TaskDone.NAME_KEY